lua/terrortown/autorun/shared/sh_pharaoh_handler.lua

```diff
@@ -358,12 +595,23 @@ if CLIENT then
 			end
 		end
 	end)
+
+	net.Receive("ttt2_net_pharaoh_can_conv", function()
+		local client = LocalPlayer()
+		client.ankh_can_conv = net.ReadInt(16)
+	end)
+
+	hook.Add("TTTBeginRound", "ttt2_role_pharaoh_reset", function()
+		local client = LocalPlayer()
```

Comment:
why cache the client here?

Reply:
ankh_can_conv is used by the client in HUDDrawTargetIDAnkh (TTTRenderEntityInfo hook for the ankh entity) to determine whether to display the "ankh_convert" message and transfer ownership progress bar when looking at an ankh.
See relevant code block here:
https://github.com/AaronMcKenney/ttt2-role_pha/blob/66b8115d828407e81e7cf3dc0c049f4aae017a70/gamemodes/terrortown/entities/entities/ttt_ankh/shared.lua#L379-L387

Explicitly, ankh_can_conv will be one of the following:
-2 if the client can't convert any ankh (ex. the client already own an ankh).
-1 if the client is able to steal any ankh (i.e. the client is a Graverobber who does not own an ankh).
Otherwise, the EntIndex of the ankh that they are able to convert (ex. the client is a Pharaoh who is looking at their stolen ankh).

ankh_can_conv is important in the rare case when there are multiple stolen ankhs present, as it can help the Pharaoh determine which of these ankhs they can steal back.